src/Cake.Common/Tools/OpenCover/OpenCoverSettingsExtensions.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.OpenCover
+{
+    /// <summary>
+    /// Contains extensions for <see cref="OpenCoverSettings"/>.
+    /// </summary>
+    public static class OpenCoverSettingsExtensions
+    {
+        /// <summary>
+        /// Adds the filter.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="filter">The filter.</param>
+        /// <returns>The same <see cref="OpenCoverSettings"/> instance so that multiple calls can be chained.</returns>
+        public static OpenCoverSettings WithFilter(this OpenCoverSettings settings, string filter)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException("settings");
+            }
+            settings.Filters.Add(filter);
+            return settings;
+        }
+
+        /// <summary>
+        /// Exclude a class or method by filter 
+        /// that match attributes that have been applied.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="filter">The filter.</param>
+        /// <returns>The same <see cref="OpenCoverSettings"/> instance so that multiple calls can be chained.</returns>
+        public static OpenCoverSettings ExcludeByAttribute(this OpenCoverSettings settings, string filter)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException("settings");
+            }
+            settings.ExcludedAttributeFilters.Add(filter);
+            return settings;
+        }
+
+        /// <summary>
+        /// Exclude a class (or methods) by filter that match the filenames.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="filter">The filter.</param>
+        /// <returns>The same <see cref="OpenCoverSettings"/> instance so that multiple calls can be chained.</returns>
+        public static OpenCoverSettings ExcludeByFile(this OpenCoverSettings settings, string filter)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException("settings");
+            }
+            settings.ExcludedFileFilters.Add(filter);
+            return settings;
+        }
+
+        /// <summary>
+        /// Sets the output file.
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="file">The output file.</param>
+        /// <returns>The same <see cref="OpenCoverSettings"/> instance so that multiple calls can be chained.</returns>
+        public static OpenCoverSettings SetOutputFile(this OpenCoverSettings settings, FilePath file)
```

Comment:
Remove this once the `OutputFile` setting has been replaced by an alias/runner argument.